### PR TITLE
resolve #7: create meta interface and package meta realization

### LINKED ISSUE #7: create meta interface and package meta realization

### PR DESCRIPTION
**Pull request by Guseyn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,11 @@ target/
 *.iml
 .DS_Store
 node_modules/
+/src/test/resources/packaging_tutorial/example_pkg/server/
```
Comment (g4s8) [inline .gitignore]: @Guseyn can you use `/src/test/resources/packaging_tutorial/` here?

Comment (Guseyn) [reply .gitignore]: @g4s8 I will remove these line later from .gitignore, so now it does not matter

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/pypi-adapter/issues/8#issuecomment-581896818))

@g4s8 This is what I know about this job in [CT2E6TK9B](https://www.0crat.com/p/CT2E6TK9B), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#8](https://github.com/artipie/pypi-adapter/issues/8) is [in scope](http://www.zerocracy.com/policy.html#14) for 2hr
  * The role is `REV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @guseyn[/z](https://www.0crat.com/u/guseyn): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fpypi-adapter%238%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@0crat assign @paulodamaso

**Comment by 0crat** [comment]
> @0crat assign @paulodamaso  ([here](https://github.com/artipie/pypi-adapter/issues/8#issuecomment-581912142))

@g4s8 This pull request [#8](https://github.com/artipie/pypi-adapter/issues/8) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/95804ecb-ab3a-4f5f-bf3e-d948a9ac5a6f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by Guseyn** [comment]
@paulodamaso hi, could you plz review my pr

**Comment by yegor256** [comment]
@0crat reject

**Comment by yegor256** [comment]
@paulodamaso I'm taking over, because it's taking too long

**Comment by yegor256** [comment]
@Guseyn thanks!

**Comment by 0crat** [comment]
> @0crat reject ([here](https://github.com/artipie/pypi-adapter/issues/8#issuecomment-582360629))

@yegor256 The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#8](https://github.com/artipie/pypi-adapter/issues/8), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#8` is not assigned, can't get performer

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**